setup/setup.iss

```diff
@@ -273,7 +279,7 @@ end;
 
 procedure CreateSelectGameExePage();
 begin
-    SelectGameExePage := CreateInputFilePage(wpSelectDir, 'Select RF.exe', 'Select Red Faction executable file', '');
+    SelectGameExePage := CreateInputFilePage(wpSelectDir, 'Select RF.exe', 'Select the Red Faction executable file', '');

```

Comment:
The installer automatically fills in this path using the locations where the game is installed to for different distributions (retail, Steam, GOG), so most users will not have to set this up manually. There is also a message above the exe path input that states `Please specify the location of RF.exe from the Red Faction installation directory:`.

Reply:
> There is also a message above the exe path input that states Please specify the location of RF.exe from the Red Faction installation directory:.

Ah ok fair enough, I missed that. My suggestion is much less relevant in that case, I still don't really see a downside to reiterating it here too though, if there's space.

Reply:
This text is used in the white header of the setup page and I think it should be pretty short. Longer description is given in the grey area and repeating it over and over IMHO only forces user to read more text without much value so I would rather keep it short and limited to a single sentence.

Reply:
Hm... What if we asked the user to select the Red Faction installation folder rather than the game executable? It would allow us to phrase the instructions in a way that is consistent with the other two pages that create Dash's folders. Then we could just find the executable file afterwards and apply the patches.

Reply:
Please don't increase scope of this PR because it already changes a lot.
Both the installation program and DF options window specifically ask for RF.exe file path instead of the root directory of the game to make it possible for more advanced users to have multiple RF.exe files with non arbitrary filenames. One may have RF 1.20, RF 1.21 Steam, 1.21 GOG, perhaps something like PF,  RF.exe with Large Address Aware patch, etc in that directory and I want to let them select the RF 1.20 NA file manually. I guess we could have separate options like RF dir and RF.exe filename that could be defaulted to RF.exe or even auto-detected by checking files checksums, but that's a big change and in fact I'm not sure it is worth the time. Current way of setting up DF is already well known and documented by community is some tutorials and I think people don't have much trouble with it.

Reply:
> One may have RF 1.20, RF 1.21 Steam, 1.21 GOG, perhaps something like PF, RF.exe with Large Address Aware patch, etc in that directory

> fundamentally changing this approach in such a way that the already-made tutorials (which have been translated into other languages as well as reposted on sites that we don't have the ability to update) could no longer be precisely followed would definitely be a mistake.

I understand. Perhaps it would be best to reword the instructions in a way that doesn't explicitly ask for `RF.exe`, as it may have been modified/renamed before installation. Sample:

<img src="https://user-images.githubusercontent.com/89315313/180675068-4d26f5c6-e216-40f9-904f-d7e91fb52d78.png" width="auto" height="300" />

Reply:
Ordinarily I'd agree with a wording change that makes the text more accurate, but I think in this case it would be best to keep the prompt specifically referencing "RF.exe". I say that because it will always be named RF.exe in the stock unmodded game - the only way it'll be something else is if it's either RF_120na.exe (which would have been created by a previous Dash install), or if the user renamed it themselves (in which case, they should know to adjust the path). I think the specific string on this prompt only really matters for new users and brand new installs, for which it will always be RF.exe.

On the flip side, changing it to "usually named RF.exe" could lead to those brand new players selecting something else (like the launcher - RedFaction.exe, or even something like PF.exe if they're using a copy of the game that has PF preinstalled as was the case for many, many distributions that were floating around years ago).

The current wording should be fine - on the whole I just think that changing the wording to "usually named" has some potential to lead to confusion for new users, while users who aren't new are very likely to know to adjust the path without the wording change.

Reply:
Here is another attempt, aimed at giving only the necessary information. Tighter and more general wording is used to make sure all users can follow along and understand what to do without reading an eyeful. Thoughts?

<img src="https://user-images.githubusercontent.com/89315313/180696679-139b736c-41c1-47c6-acf1-0116b188cf2a.png" width="auto" height="200" />

Reply:
That new wording seems good to me. Do you like it rafalh?

Reply:
Well, idk, it does not matter that much... I was thinking of those captions on white background as a title and subtitle but it's actually caption and description so maybe it makes sense.
But I would prefer to bring back the input field "prompt" (text right before input field) because now we have some information about RF.exe detection but there is no label for the input itself (e.g. "RF.exe path:").
Also I'm not sure if it really improves anything if we tell people what clicking Next and Browse does. Isn't it obvious?
But I don't have strong opinion here (except maybe for the prompt) so it may stay like this.

Reply:
I can add the input field text back in, I don't see why not. I added the click next text for consistency, as both shortcut pages and the task page in the setup include the same text by default.